docs/reference/api-v3.yml

```diff
@@ -225,6 +225,83 @@ paths:
           - an object sent in the packagings field will replace any pre-existing data.
           - an object sent in the field suffixed with _add (e.g. packagings_add) will be merged with any pre-existing data.
       parameters: []
+  /api/v3/taxonomy_suggestions:
```

Comment:
It could be interesting to add a param where if the parameter is present, in the response we not only send back the tag value but also it's id. (eg. with_id=1).

This might be usefull for hunger-games or other tools.

Reply:
Yes, it makes things a bit more complex to describe in openapi as in that case we need to return an array of objects instead of an array of strings, but I think it's possible.